### ISSUE TITLE
ImageBind微调

### ISSUE THREAD

**Issue by starstar1-J** [jasongief/OV-AVEL#7]
作者您好！

> 我注意到项目中提供的 ImageBind 相关代码并非官方原始版本，因此想确认：该版本是否已针对本项目的目标数据集做过针对性微调？

> 如果做了微调，想进一步确认：是在seen和unseen上都参与过微调吗？具体是怎么做的？

期待您的回复！

**Comment by jasongief** [comment]
你好，感谢你对这个工作的关注。我想你对代码可能有误解？如果我没记错，这个 ImageBind 相关代码 和 权重 应该都是[官方](https://github.com/facebookresearch/ImageBind)原始版本，官方 release 的权重也只有一个？
ImageBind 的 encoder 应该是参数frozen 的，论文提供了两个 baseline，training-free 的版本没有微调任何参数，直接在 seen 和 unseen test set 测试（zero-shot）。对于fine-tuning 的 baseline版本， 其添加了 temporal layer，对应的参数会被 fine-tune。两个版本的 baseline 中的 ImageBind 应该都是 fix 的，不然要求的显存比较高。
